source/stylesheets/refills/_search-tools.scss

```diff
@@ -82,16 +82,21 @@
       @include position(absolute, 5px 5px auto auto);
       font-size: 0.7em;
     }
+  }
 
-    .hide-options {
-      margin-bottom: 0;
 
-      ol, button {
-        display: none;
-      }
+  .hide-options {
+    margin-bottom: 0;
+
+    ol, button {
+      display: none;
     }
   }
 
+  .summary {
+    padding-right: 4.5em;
```

Comment:
Is this a [magic number](https://css-tricks.com/magic-numbers-in-css/)? If so, can it be avoided?